docs/span_attribute_schema/v1.md

```diff
@@ -238,7 +238,25 @@ grpc.method.package | Yes
 grpc.method.path | Yes
 grpc.method.service | Yes
 grpc.status.code | Yes
-span.kind | `client`; `server`
```

Comment:
shouldn't we have `out.host`, `peer.service` and `_dd.peer.service.source` listed here now as well?

Reply:
Fixed in 8974133ef6a34eac20fdfc24028fa8e98f3711f4